sda/simulation_test.go

```diff
@@ -43,7 +47,12 @@ func TestSimulationBF(t *testing.T) {
 	}
 }
 
-func TestBigTree(t *testing.T) {
+func TestSimulationBigTree(t *testing.T) {
+	t.Skip()
```

Comment:
Yep it's longer to write when testing but i've put it back. I still don't know what is it useful for ? It really takes **seconds** each time it runs for something quite deterministic. I mean can't we reduce the number of nodes generated to get faster tests ?